### PR TITLE
Content Management - Filter out list already exisiting and add type to list object structure

### PR DESCRIPTION
**Pull request by daftops**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
When managing XSOAR lists with the Content Management pack, default behavior is to create or overwrite lists defined in xsoar_config.json. With this modification an user can select to avoid overwriting of an existing list by setting the key "type" in the List object of xsoar_config.json to "dynamic". 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
please add documentation about the lists options(created or not created from the xsoar_config.json) using the keys.
This is the suitable spot I believe - https://xsoar.pan.dev/docs/reference/packs/content-management#playbook

**Comment by adi88d** [comment]
@daftops Which company are you working for?

**Review by adi88d — APPROVED** [review]
Nice

**Comment by daftops** [comment]
> Nice

@adi88d Upstart

File: Packs/ContentManagement/Scripts/ConfigurationSetup/ConfigurationSetup.py [hunk 1]
```diff
@@ -208,8 +208,20 @@ def load_lists(self) -> None:
...
+                list_type = _list.get('type')
 
-                self.lists[list_name] = list_value
+                self.lists[list_name] = {
+                    "value": list_value,
+                    "type": list_type
+                }
+
+
+def exist_list(list_name: str) -> bool:
```
Comment (tkatzir) [inline Packs/ContentManagement/Scripts/ConfigurationSetup/ConfigurationSetup.py:219]:
```suggestion
def list_exists(list_name: str) -> bool:
```

File: Packs/ContentManagement/Scripts/ConfigurationSetup/ConfigurationSetup.py [hunk 2]
```diff
@@ -236,9 +248,9 @@ def create_context(full_configuration: Configuration) -> Dict[str, List[Dict[str
     lists = [
         {
             'listname': list_name,
-            'listdata': list_data,
+            'listdata': pps["value"],
         }
-        for list_name, list_data in full_configuration.lists.items()
+        for list_name, pps in full_configuration.lists.items() if not exist_list(list_name) or pps["type"] != "dynamic"
```
Comment (tkatzir) [inline Packs/ContentManagement/Scripts/ConfigurationSetup/ConfigurationSetup.py:253]:
```suggestion
        for list_name, pps in full_configuration.lists.items() if not list_exists(list_name) or pps["type"] != "dynamic"
```

**Pull request merged**